src/plugins/SysML/SysMLExporter/Backup Templates/Block.uml.ejs

```diff
@@ -0,0 +1,4 @@
+  <uml:Model xmi:id="_XwW94JQEEeWSjcRKbkPmoQ" name="RootElement">
```

Comment:
Are these Backup Templates used?
(In general it's good practice to avoid spaces and special characters in paths..)